docs/sphinx/reference-libobs-util.rst

```diff
@@ -8,6 +8,7 @@ Platform/Utility API Reference (libobs/util)
    reference-libobs-util-bmem
    reference-libobs-util-circlebuf
```

Comment:
Probably makes sense to remove this as well then.

Reply:
If it's still in available in the API, it doesn't make sense to remove it even if it's "deprecated".
But the whole page should been updated to have all the function shown as deprecated, which is out of the scope that I want to keep for this PR.

Reply:
@derrod Seems resolved?